.github/workflows/codecov.yml

```
@@ -13,6 +13,8 @@ jobs:
     environment: CI Environment
     steps:
       - uses: actions/checkout@v2 
+        with:
+              submodules: recursive
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 14

(indentation)